doc/code_snippets/test/sql/foreign_key_table_constraint_test.lua

```diff
@@ -0,0 +1,82 @@
+local fio = require('fio')
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group()
+g.before_each(function(cg)
+    cg.server = server:new {
+        box_cfg = {},
+        workdir = fio.cwd() .. '/tmp'
+    }
+    cg.server:start()
+end)
+
+g.after_each(function(cg)
+    cg.server:stop()
+    cg.server:drop()
+end)
+
+g.test_space_is_updated = function(cg)
+    cg.server:exec(function()
+        box.execute([[SET SESSION "sql_seq_scan" = true;]])
+        box.execute([[
+            -- create_parent_start
+            CREATE TABLE author (
+                id INTEGER PRIMARY KEY,
+                name VARCHAR(100) NOT NULL
```

Comment:
I suggest using `STRING` instead of `VARCHAR(...)` here and below, since our SQL does not support fixed length string values and will automatically replace them with `STRING`.